commit 2dcf0926572589ac52e3410a02c7ffcada440fad
Author: Jonathan Riddell <jr@jriddell.org>
Date:   Tue Apr 28 13:45:36 2015 +0200

    fix compile by finding kf5 iconthemes

Index: libqapt-3.0.0/CMakeLists.txt
===================================================================
--- libqapt-3.0.0.orig/CMakeLists.txt
+++ libqapt-3.0.0/CMakeLists.txt
@@ -75,6 +75,7 @@ find_package(KF5Runner ${KF5_DEP_VERSION
 find_package(KF5TextWidgets ${KF5_DEP_VERSION})
 find_package(KF5WidgetsAddons ${KF5_DEP_VERSION})
 find_package(KF5WindowSystem ${KF5_DEP_VERSION})
+find_package(KF5IconThemes ${KF5_DEP_VERSION})
 
 find_package(DebconfKDE)
 find_package(GStreamer)
@@ -87,6 +88,7 @@ if (KF5CoreAddons_FOUND
         AND KF5TextWidgets_FOUND
         AND KF5WidgetsAddons_FOUND
         AND KF5WindowSystem_FOUND
+        AND KF5IconThemes_FOUND
         AND DebconfKDE_FOUND)
     set(WITH_UTILS true)
 endif()
